README.md

```diff
@@ -1,6 +1,17 @@
 # CSI NFS driver
 
-## Kubernetes
+## Overview
+
+This is a repository for [NFS](https://en.wikipedia.org/wiki/Network_File_System) [CSI](https://kubernetes-csi.github.io/docs/) Driver.
+Currently it implements bare minimum of the CSI and the state is alpha.
+
+#### CSI Feature matrix
+
+| **nfs.csi.k8s.io** | CSI versions compatibility| Dynamic Provisioning | Resize | Snapshots | Raw Block | AccessModes              | Status |
+|--------------------|---------------------------|----------------------|--------|-----------|-----------|--------------------------|--------|
+|master              | v1.0.0, v1.1.0            |  no                  |  no    |  no       |  no       | Read/Write Multiple Pods | Alpha  |
+|v1.0.0              | v1.0.0, v1.1.0            |  no                  |  no    |  no       |  no       | Read/Write Multiple Pods | Alpha  |
+
 ### Requirements
 
 The folllowing feature gates and runtime config have to be enabled to deploy the driver
```

Comment:
I'm unsure if all the feature gates and alpha runtime config below are necessary anymore.

Reply:
Also I think we should remove the csc tool instructions. I don't think it's been maintained for a while now.